File: code-examples/java/coupons/delete.java
HttpResponse<String> response = Unirest.delete("http://api.marketcloud.it/v0/coupons/:id")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .header("authorization", "YourPublicKey:YourToken")
  .asString();

File: code-examples/csharp/coupons/delete.cs
var client = new RestClient("http://api.marketcloud.it/v0/coupons/:id");
var request = new RestRequest(Method.DELETE);
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
IRestResponse response = client.Execute(request);